hazelcast/src/main/java/com/hazelcast/jet/Job.java

```diff
@@ -360,4 +361,23 @@ default String getName() {
      * @throws UnsupportedOperationException if called for a light job
      */
     JobStateSnapshot exportSnapshot(String name);
+
+    /**
+     * Associates the given listener to this job. The listener is automatically
+     * removed after a {@linkplain JobStatus#isTerminal terminal event}.
+     *
+     * @return The registration id
+     * @throws UnsupportedOperationException if the cluster version is less than 5.3
+     * @since 5.3
+     */
+    UUID addStatusListener(@Nonnull JobStatusListener listener);
```

Comment:
these methods are below comment: `// ### Methods below apply only to normal (non-light) jobs.`

Reply:
they are still below the comment